dockerfiles/integration/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# This Dockerfile is only for running integration tests that depend on a running containerd process
```

Comment:
nit: from what I understood, it's supposed to not only running `integration` tests, but also `testflight`. 

```suggestion
 # This Dockerfile is only for running testflight and other integration tests
```